restapi/impl/db/permissions.go

```diff
@@ -215,6 +239,63 @@ func PermissionsForSubjectsAndResourceTypeMinLevel(
 	return rowsToPermissionList(rows)
 }
 
+// permissionLevelPrecedenceExpression returns a SelectBuilder representing a permission level precedence
+// expression that can be used in a where clause.
+func permissionLevelPrecedenceExpression(prefix, permissionLevel string) sq.SelectBuilder {
+	return psql.Select("precedence").
+		Prefix(fmt.Sprintf("%s (", prefix)).
+		From("permission_levels").
+		Where(sq.Eq{"name": permissionLevel}).
+		Suffix(")")
+}
+
+// AbbreviatedPermissionsForSubjectAndResouorceType lists permissions for a subject and resource type. If the
+// minLevel parameter is specified, permissions that don't meet or exceed the minimum level will be omitted
+// from the results.
+func AbbreviatedPermissionsForSubjectAndResouorceType(
```

Comment:
typo:

```suggestion
func AbbreviatedPermissionsForSubjectAndResourceType(
```
(and presumably the docs and anywhere that calls this, of course)

Reply:
Thanks for catching that!

Reply:
I'm fixing this one locally just so that my code editor can handle editing the docs and all of the function calls for me. 😁